docs/conf.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
```

Comment:
We don't tend to embed the docs in the project themselves but make centrazlied at https://docs.chaostoolkit.org/drivers/overview/. Maybe we could simply remove sphinx here?